.github/workflows/publish-release.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           key: ${{ github.sha }}
       - name: Dry Run Publish
         # omit npm-token token to perform dry run publish
-        uses: MetaMask/action-npm-publish@v4
+        uses: MetaMask/action-npm-publish@v1
```

Comment:
We'll need to update all references to this action, there is one more below in the `publish-npm` step